### ISSUE TITLE
7.9.1 Problems on MacOS Sierra - Updated Version Dialog Box

### ISSUE THREAD

**Issue by jcferguson** [LMS-Community/slimserver#177]
I'm running 7.91 - 1509357021 on MacOS 10.12.6, connecting to five RPi 3 devices all running PiCorePlayer v3.22. Three are connected via Ethernet and two via wireless. 

After upgrading from 7.7 to 7.91 on MacOS 10.12.6, I continue to get the message, "An updated [LMS] version is available and ready to be installed. Clicking on "Install Update" seems to cause nothing to happen. It also goes away if clicking "Not Now" but in either case, pops up again in a few minutes. If viewing the web server management interface, it also indicates an update is available. 
I've done a clean install a couple of times, but the behavior is consistent.

This is two of three problems.

**Comment by mherger** [comment]
What build 7.9.1 do you have installed? There can be daily new builds.

Also check your cache folder (see Settings/Information). There's an Updates folder where the downloaded installer would be stored. What's its file name?

**Comment by jcferguson** [comment]
Build 7.9.1-1509357021 is installed. The version in the .../Caches/Squeezebox folder is 7.9.1-1510467881.pkg. 
I'll wait on attempting to install via the .pkg file until the other tasks are completed.

**Comment by mherger** [comment]
Quite obviously the message is valid: there is a newer version available, it's downloaded and you can install it.
